products/jbrowse-web/src/tests/TextSearching.test.tsx

```diff
@@ -1,13 +1,18 @@
 // library
 import React from 'react'
-import { cleanup, waitFor, fireEvent, render } from '@testing-library/react'
+import {
+  screen,
+  cleanup,
+  waitFor,
+  fireEvent,
+  render,
+} from '@testing-library/react'
 import { LocalFile } from 'generic-filehandle'
 
 // locals
 import { clearCache } from '@jbrowse/core/util/io/rangeFetcher'
 import { clearAdapterCache } from '@jbrowse/core/data_adapters/dataAdapterCache'
 import { setup, generateReadBuffer, getPluginManager } from './util'
-import jb1_config from '../../test_data/volvox/volvox_jb1_text_config.json'
```

Comment:
Are we no longer testing the use of JB1 text search adapter ?

Reply:
We will want to add back some tests for it, they were not yet functional re the prefix vs exact

Reply:
Tested the branch, looks good and returns the expected options but did we want to redesign the flow of prefix vs exact?

Reply:
As far as I can tell, the workflow is right on the trix side of things so it might come back to the jb1 adapter behavior